.github/workflows/ci.yaml

```diff
@@ -2,6 +2,18 @@ name: CI
 
 on:
   pull_request:
+    paths-ignore:
+      - '.github/**'
+      - 'configsamples/**'
+      - 'design/**'
+      - 'website/**'
```

Comment:
The website is validated against CI (specifically, there's a "validate-website-build" lane that validates if hugo is able to build the website).

So we either run all the CI or just that lane when the content of website change

Reply:
Bumped the site-build out into its own workflow so that it is only triggered by the `website/**` and `netlify.toml` paths